app/controllers/matches_controller.rb

```diff
@@ -1,10 +1,13 @@
 class MatchesController < ApplicationController
-  before_action :active_staff, :same_organization?
+  before_action :set_pet, only: %i[create]
+  before_action :set_match, only: %i[destroy]
```

Comment:
Are these lines duplicated on lines 5-6?

Reply:
Not sure how that happened